### PR TITLE
basic files creation and deletion. .editconfig added. react-redux and…

### PR DESCRIPTION
**Pull request by cornets28**
Book Management App
- create-react-app created the basic files
- Redux and react-redux was added
- .editorcnfig file was added

### CODE REVIEW

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @CornetS28 ,

Great job setting up the project.

However, we need to take care of some things before moving on:
- `redux` and `react-redux` should be under `dependencies` instead of `devDependencies` as they are required at runtime not just during development.
- Let's set up Stickler on the repo. Instructions [here](https://github.com/microverseinc/linters-config/tree/master/react-redux).
- We can remove unnecessary files which are not needed for this project, like `serviceWorker.js` and also `App.test.js` unless you plan to write tests for the project.
- Please add a descriptive readme

**Please update the app and submit another review request.**

**Comment by cornets28** [comment]
> Hi @CornetS28 ,
> 
> Great job setting up the project.
> 
> However, we need to take care of some things before moving on:
> 
> * `redux` and `react-redux` should be under `dependencies` instead of `devDependencies` as they are required at runtime not just during development.
> * Let's set up Stickler on the repo. Instructions [here](https://github.com/microverseinc/linters-config/tree/master/react-redux).
> * We can remove unnecessary files which are not needed for this project, like `serviceWorker.js` and also `App.test.js` unless you plan to write tests for the project.
> * Please add a descriptive readme
> 
> **Please update the app and submit another review request.**

Thank you for your feedback!

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+### `yarn start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### `yarn test`
+
```
Comment (SunnyAsar) [inline README.md]: please clean up this file. let your `readme` be project specific and not the default provided by react. Add things like the setup info, contributor(s) etc.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "book-management",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-route-hook": "^1.2.1",
```
Comment (SunnyAsar) [inline package.json]: we don't need some of these for this project. here are the requirements from pathwrite.

AddReduxAndReact-Redux

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @CornetS28 👋 
Well done with the project setup, for the finish, please install the required packages.

**Please request another review with the update**

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
```
Comment (SunnyAsar) [inline src/index.js]: let's fix the linting errors.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @CornetS28 
Please check the linting errors in the code and make corrections.

**Please request another review when done**

**Review by shloch — APPROVED** [review]
## Nice job, @CornetS28 

for next milestones, make branch names to be descriptive, something like `milestone-2-addingDisplay` 

### You can merge this PR 👏👏👏

**Pull request merged**